sdk/communication/azure-communication-phonenumbers/test/recordings/test_phone_number_administration_client.pyTestPhoneNumbersClienttest_update_phone_number_capabilities_with_invalid_number.json

```diff
@@ -1,15 +1,15 @@
 {
   "Entries": [
     {
-      "RequestUri": "https://sanitized.communication.azure.com/phoneNumbers/invalid_phone_number/capabilities?api-version=2022-01-11-preview2",
+      "RequestUri": "https://sanitized.communication.azure.com/phoneNumbers/invalid_phone_number/capabilities?api-version=2022-12-01",
```

Comment:
While the change in the request makes sense, it worries me that they are using different API versions. This makes me think that there is a possibility that the recordings for the other tests are using different versions, which shouldn't be the case.